src/goose/api/batch.clj

```diff
@@ -0,0 +1,21 @@
+(ns goose.api.batch
+  (:require [goose.broker :as b]
```

Comment:
nitpick: this isn't alphabetically ordered, or formatted like rest of goose code.

Enabling format-on-save or are running auto-format before committing should solve such issues.

Reply:
Ordered it. 

I've intentionally disabled import optimisation on save, since it was not as per the convention we follow for Goose  (separating Goose dependencies and Clojure dependencies).